src/words/ast/INodeQueueSay.java

```
@@ -31,6 +32,9 @@ public ASTValue eval(WordsEnvironment environment) throws WordsRuntimeException
 		} else {
 			object = environment.getObject(identifier.stringValue);
 		}
+		if (object == null) {
+			throw new WordsObjectNotFoundException(identifier.stringValue);
```

WordsEnvironment.getObject() should throw this error if the object isn't found.